packages/headless/src/entity/compare.ts

```diff
@@ -1,11 +1,19 @@
 export class IProductInfo {
   name: string;
   price: number;
+  oldPrice: number;
   shortDescription: string;
   fullDescription: string;
 }
+export class IProductMeta{
```

Comment:
We need just one property(friendlyPageName).

Reply:
please elaborate